frost-ristretto255/README.md

```diff
@@ -3,30 +3,98 @@ of signers (FROST).
 
 ## Examples
 
-Creating a `Signature` with a single signer, serializing and deserializing it, and verifying the
-signature:
+### Trusted Dealer
+
+Creating a key with a trusted dealer and splitting into shares; then signing a message
+and aggregating the signature. Note that the example just simulates a distributed
+scenario in a single thread and it abstracts away any communication between peers.
+
 
 ```rust
+use std::{collections::HashMap, convert::TryFrom};
 use rand::thread_rng;
-use frost_ristretto255::*;
+use frost_ristretto255 as frost;
+
+let mut rng = thread_rng();
+let numsigners = 5;
+let threshold = 3;
```

Comment:
I need to change this to min/max_signers and then also copy the example to #164 after that merges so I'll wait for that to happen and update this PR

Reply:
Done